src/style.css

```diff
@@ -6,10 +6,16 @@ body {
 .all {
   border: solid;
   margin: 20px;
+  background-color: rgb(65, 117, 147);
 }
 
 .maintitle {
-  margin: 10px;
+  text-align: center;
+  border: solid;
+  margin: 20px;
+  border-radius: 10px;
+  background: #fff;
+  box-shadow: 3px 3px 2px rgb(11, 11, 11);
 }
 
```

Comment:
- [ ] Great job with the animation. Let's try to maintain the layout from the wireframe. Kindly align the refresh button to the right side. I think it would be great if the submit button is aligned to the right side to match the wireframe. 😎️

**Sample 1**
<img src="https://user-images.githubusercontent.com/17243325/216653234-1f2d5304-b2e5-47d7-8cb4-cc7b5d8940c4.png" width="400">

**Sample 2**
<img src="https://user-images.githubusercontent.com/17243325/216653295-23d59a3a-713f-4795-b13d-bc5eb228b69c.png" width="400">